projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/asm-b2b-customer-list-tabbling-order.e2e.cy.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { tabbingOrderConfig as config } from '../../helpers/accessibility/tabbing-order.config';
+import { asmTabbingOrderWithCustomerList } from '../../helpers/accessibility/tabbing-order/asm';
+import { ELECTRONICS_BASESITE } from '../../helpers/checkout-flow';
+import { POWERTOOLS_BASESITE } from '../../sample-data/b2b-checkout';
+
+context('Tabbing order for ASM B2b Customer List', () => {
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+    Cypress.env('BASE_SITE', POWERTOOLS_BASESITE);
```

Comment:
setting basesite maybe unnecessary, here is a suitable command,please see: https://github.com/SAP/spartacus/blob/8fa9315dcae04544a753719f1f88f05b4682ab52/projects/storefrontapp-e2e-cypress/package.json#L11